prisma/schema.prisma

```diff
@@ -164,17 +165,17 @@ enum UserRole {
 }
 
 model User {
-  id            String       @id @default(cuid())
-  name          String?
-  email         String?      @unique
-  emailVerified DateTime?
-  image         String?
-  role          UserRole?    @default(USER)
-  accounts      Account[]
-  sessions      Session[]
-  resources     Resource[]  @relation("Resources")
+  id              String       @id @default(cuid())
```

Comment:
I'm wondering where these changes came from? I'm surprised to see them

Reply:
Ooh you are right, i didnt expect to see them either. honestly, I couldnt quite tell you

Reply:
thanks for the tip